ggml/src/ggml-cuda/out-prod.cu

```diff
@@ -0,0 +1,52 @@
+#include "out-prod.cuh"
+#include "vendors/cuda.h"
```

Comment:
```suggestion
```

Reply:
This should do it, this file cannot be included directly, and it is already included in `common.cuh`.

Reply:
I can confirm that this fixes compilation with `GGML_HIPBLAS`.